src/docfx/build/xref/ExternalXrefMapLoader.cs

```diff
@@ -12,13 +12,15 @@ namespace Microsoft.Docs.Build
 {
     internal class ExternalXrefMapLoader
     {
+        private const string RepoUrlKey = "repositoryUrl";
+
         private static readonly byte[] s_uidBytes = Encoding.UTF8.GetBytes("uid");
 
-        public static IReadOnlyDictionary<string, Lazy<ExternalXrefSpec>> Load(Config config, FileResolver fileResolver, ErrorBuilder errors)
+        public static IReadOnlyDictionary<string, Lazy<(string?, ExternalXrefSpec)>> Load(Config config, FileResolver fileResolver, ErrorBuilder errors)
```

Comment:
can we add repourl to ExternalXrefSpec model, like `from`?

Reply:
so we don;t need change so many interfaces?

Reply:
OK